repose-aggregator/components/filters/header-normalization-filter/src/test/scala/org/openrepose/filters/headernormalization/HeaderNormalizationFilterTest.scala

```diff
@@ -48,23 +50,29 @@ class HeaderNormalizationFilterTest extends FunSpec with BeforeAndAfterEach with
   var filter: HeaderNormalizationFilter = _
   var servletRequest: MockHttpServletRequest = _
   var servletResponse: MockHttpServletResponse = _
-  var filterChain: FilterChain = _
-  var filterConfig: FilterConfig = _
-  var metricsService: MetricsService = _
-  var metricsMeters: MeterByCategorySum = _
+  private val filterChain = mock[FilterChain]
```

Comment:
I honestly don't know how or why this change got in here. I've actually been trying to go the other way everywhere I see `reset`. I agree that creating new instances per test is the way to go.

I fix.